### DISCUSSION TITLE
Step 3. Understanding of Protobuf & gRPC

### DISCUSSION THREAD

**Discussion by choonho** [cloudforet-io/community#138] [Mentorship]
Protobuf & gRPC is the base framework of Cloudforet.

# Agenda

* [Introduction to Protocol Buffers](https://training.linuxfoundation.org/training/introduction-to-protocol-buffers-lfs145/)
* What is gRPC?
  * [What is gRPC? Protocol Buffers, Streaming, and Architecture Explained](https://www.freecodecamp.org/news/what-is-grpc-protocol-buffers-stream-architecture/)
  * [Introduction to gRPC](https://grpc.io/docs/what-is-grpc/introduction/)
  * [Core concepts, architecture and lifecycle](https://grpc.io/docs/what-is-grpc/core-concepts/)
  * [Python/Quick start](https://grpc.io/docs/languages/python/quickstart/)
  * [Python/Basics tutorial](https://grpc.io/docs/languages/python/basics/)

# Submission

* Certification of Learning (Introduction to Protocol Buffers)

**Comment by Cvr421** [comment]
@choonho i have uploaded the LFX certification of Introduction to Protocol Buffers

**Reply by Tyagiquamar** [reply]
@choonho I have uploaded the LFX certificate of Introduction to Protocol Buffers

**Comment by arup1221** [comment]
@choonho   I have uploaded the LFX certification of Introduction to Protocol Buffers (LFS145)

**Reply by choonho** [reply]
Great Job. Knowledge of Protobuf & gRPC will be helpful in your career.

**Reply by arup1221** [reply]
Thank You !
